gpx.go

```diff
@@ -83,10 +83,10 @@ type MetadataType struct {
 	Author     *PersonType     `xml:"author,omitempty"`
 	Copyright  *CopyrightType  `xml:"copyright,omitempty"`
 	Link       []*LinkType     `xml:"link,omitempty"`
-	Time       time.Time       `xml:"time,omitempty"`
+	Time       string          `xml:"time,omitempty"`
```

Comment:
This change breaks backwards compatibility, which will require a major version bump.

What's the reason for this change?